src/site/content/en/blog/optimizing-content-efficiency-eliminate-downloads/index.md

```diff
@@ -23,9 +25,38 @@ The fastest and best-optimized resource is a resource not sent. You should elimi
 Too often, pages contain resources that are unnecessary, or worse, that hinder page performance without delivering much value to the visitor or to the site they're hosted on. This applies equally to first-party and third-party resources and widgets:
 
 - Site A has decided to display a photo carousel on its homepage to allow the visitor to preview multiple photos with a quick click. All of the photos are loaded when the page is loaded, and the user advances through the photos.
-  - **Question:** Have you measured how many users view multiple photos in the carousel? You might be incurring high overhead by downloading resources that most visitors never view.
+  - **Question:** Have you measured how many users view multiple photos in the carousel? You might be incurring high overhead by downloading resources that most visitors never view. This is a key scenario in which lazy loading—[images in this case](/browser-level-image-lazy-loading/)—can be used effectively to reduce overall load time and divert much needed bandwidth to other key resources on the page that the user is more likely to benefit from.
```

Comment:
Oh this is more complicated for horizontal images. See: https://bugs.chromium.org/p/chromium/issues/detail?id=1391989 so might want to leave this extra line out.

Reply:
Good call. I can strip it out.